### PR TITLE
Fix ovn script so that apiserver can be fetched correctly

### PR DESCRIPTION
**Pull request by rajatchopra**
use go-template format instead

Signed-off-by: Rajat Chopra <rchopra@redhat.com>

### CODE REVIEW

**Comment by rajatchopra** [comment]
Tested ok manually: dind with OVN.

**Comment by rajatchopra** [comment]
@stevekuznetsov PTAL. Thanks.

**Comment by rajatchopra** [comment]
@knobunc @JacobTanenbaum PTAL

**Comment by stevekuznetsov** [comment]
Why do we still need the `| grep http`?

**Comment by imcsk8** [comment]
LGTM

File: images/dind/master/ovn-kubernetes-master.sh [hunk 1]
```diff
@@ -15,7 +15,7 @@ function ovn-kubernetes-master() {
 
   local master_config="${config_dir}/master-config.yaml"
   cluster_cidr=$(python -c "import yaml; stream = file('${master_config}', 'r'); y = yaml.load(stream); print y['networkConfig']['clusterNetworks'][0]['cidr']")
-  apiserver=$(oc --config="${kube_config}" config view -o custom-columns=server:clusters[0].cluster.server | grep http)
+  apiserver=$(oc --config="${kube_config}" config view -o go-template='{{ (index .clusters 0).cluster.server }}' | grep http)
```
Comment (pravisankar) [inline images/dind/master/ovn-kubernetes-master.sh]: As @stevekuznetsov suggested, '| grep http' seems unnecessary.

**Comment by rajatchopra** [comment]
@pravisankar, @stevekuznetsov  Fixed. PTAL. Thanks.

**Pull request merged**